darkmoon/api/v1/metadata/views.py

```diff
@@ -355,6 +403,47 @@ async def hash_comparison(
         sha256Hash = h_sha256.hexdigest()
         sha512Hash = h_sha512.hexdigest()
 
+        obj = MetadataEntity(
```

Comment:
use the get_metadata() method from utils.py to gather and fill in all the information for the input file rather than leaving these sections blank.

Reply:
We are working on this, but we are going to do it as part of the ticket to convert the hashComparison endpoint to use utils.py instead of as part of this ticket.